src/hello_client.erl

```diff
@@ -165,12 +166,16 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{waiting_for_pong = true, keep_alive_interval = KeepAliveInterval}) ->
+    ?LOG_ERROR("There is no PONG answer on PING for ~p msec. Client will be restarted.", [KeepAliveInterval]),
```

Comment:
@skattelman, I disagree little bit, that it is a job of hello to know, what time the connection was dropped. The connection should write some metric, like alive or not to exometer and some other aggregation system can do graph it and take the time, where it happens first time. There are ready and simple tools to do it, to not make from hello complex, where it possible to reuse existing tools for the same reasons.